src/utils.js

```diff
@@ -0,0 +1,15 @@
+findHighest = (a,b,c) => {
+    let highest = 0;
+    if(a > b) {
+      highest = a;
+    }else {
+      highest = b;  
+     }
+    if(c > highest)
+      highest = c;
+    
+    return highest;    
```

Comment:
What happens if we have five numbers we want to check. How will this logic look? Can you figure out a much better way to find the highest

Reply:
I will